translations/harbour-matkakortti-zh_CN.ts

```diff
@@ -0,0 +1,167 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE TS>
+<TS version="2.1">
```

Comment:
This should be `<TS version="2.1" language="zh_CN">`. I can do it myself though, not a bit deal.